launcher/image/test/create_gpu_vm.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+local OPTIND
```

Comment:
I don't understand why this needs to be a separate script rather than extending the existing util.